packages/model-viewer/src/features/scene-graph/api.ts

```diff
@@ -32,7 +32,7 @@ export declare interface ThreeDOMElementMap {
 }
 
 /** A 2D Cartesian coordinate */
-export interface Vector2 {
+export interface Vector2D {
```

Comment:
Okay, so now instead of shadowing three's construct, you're shadowing our existing public class from model-viewer-base. Any reason not to use `Vector2DInterface` like you did before?

Reply:
Not really, I think this is trying to be an interface for threejs's vector2, so maybe Vector2Interface is better?

Reply:
Sure, SG